lib/load.php

```diff
@@ -121,6 +121,11 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/global-styles.php';
 require __DIR__ . '/pwa.php';
 
+// TODO: Before this PR merges, move this to be a part of the style engine package.
```

Comment:
Shall we update this comment slightly?

```suggestion
// TODO: Move this to be loaded from the style engine package, via the build directory.
```